docs/source/api/cdns_id_snapshot.rst

```diff
@@ -25,6 +25,8 @@
 =======
 Performs a CDN snapshot. Effectively, this propagates the new *configuration* of the CDN to its *operating state*, which replaces the output of the :ref:`to-api-cdns-name-snapshot` endpoint with the output of the :ref:`to-api-cdns-name-snapshot-new` endpoint.
 
+.. Note:: Snapshotting the CRConfig also deletes all HTTPS certificates for delivery services which have been deleted since the last snapshot.
```

Comment:
Please use "Snapshot" or "CDN Snapshot" as opposed to "CRConfig" (not a big deal in the TO using docs, because that whole thing is deprecated/disabled. Frankly I wouldn't care if you didn't update it). It's probably less awkward to refer to the action as "taking a CDN Snapshot" rather than "Snapshotting the CDN Snapshot".

Also, "Snapshot" and "Delivery Service" are ATC *terms* and should be linked to the glossary. At a minimum that should be done once per major section ("page" in an HTML context) but ideally it'd be all of them. Regardless of whether you decide to link them here - assuming it is done elsewhere in the same major section - they should be capitalized like that for ease of recognition.

Reply:
E.g. term usage

```rst
This -> :term:`Delivery Service` will link to the glossary for the definition of a DS.
For the moment, the glossary has a problem with plurals, so to link the plural of
a term to the glossary, link the singular form and place an 's' with an escaped
space after the closing backtick, e.g. :term:`Delivery Service`\ s.
```

Reply:
>link the singular form and place an 's' with an escaped space after the closing backtick, e.g.
:term:`Delivery Service`\ s.

Tangent: How terrible would it be to recommend using zero-width spaces, so it looks right?
https://www.fileformat.info/info/unicode/char/200B/index.htm

Reply:
>use "Snapshot" or "CDN Snapshot" as opposed to "CRConfig"

>"Snapshot" and "Delivery Service" are ATC _terms_ and should be linked

Done.

Reply:
A better solution than zero-width spaces is just adding to the list of terms defined by the "Delivery Service" glossary entry. right now we have something like:

```rst
.. glossary::

    Delivery Service
        A :dfn:`Delivery Service` is a thing that ....
```

... but the glossary would support:

```rst
.. glossary::

    Delivery Service
    Delivery Services
        A Delivery Service is a thing that....
```

Once that's done:

```rst
Both :term:`Delivery Service` and :term:`Delivery Services` link to the glossary appropriately.
```

The only reasons that isn't done already are because
1) I didn't know you could do that at the time I added the Delivery Service glossary entry
2) I haven't gotten around to changing it yet. Feel free to roll that into this PR if you want - though preferably as a separate commit.

Reply:
Meh, I just reworded it to be singular.

Reply:
I don't blame you.